samples/aws-codestar-lambda/project.clj

```diff
@@ -8,7 +8,7 @@
                  [io.pedestal/pedestal.jetty "0.5.5"]
                  [io.pedestal/pedestal.aws "0.5.5" :exclusions [joda-time]]
 
-                 [ch.qos.logback/logback-classic "1.2.3" :exclusions [org.slf4j/slf4j-api]]
+                 [ch.qos.logback/logback-classic "1.2.10" :exclusions [org.slf4j/slf4j-api]]
                  [org.slf4j/jul-to-slf4j "1.7.25"]
```

Comment:
Consider bumping the `org.slf4j/*` libs to `1.7.35` as well?

Reply:
Makes sense in this context, I guess.